### DISCUSSION TITLE
Using next-i18next outside react component

### DISCUSSION THREAD

**Discussion by adarshaacharya** [i18next/next-i18next#2204] [Q&A]
One of the frustating thing from past 2-3 weeks while adding i18 on my project is I can't do translation outside the React component.
Since `next-i18next` provides `useTranslation` hook which can be obviously used inside recat component but is there any way I can use it on my consts file or utils file which aren't react component.

I've tried few solutions on internet but none of them worked. :(

**Comment by adrai** [comment]
There is no "good" way to use i18next outside...
https://github.com/i18next/next-i18next/issues/2079#issuecomment-1386970151

**Reply by adarshaacharya** [reply]
@adrai any workaround at least i dont want o put everything inside my component tree, :( at least messes the cleancode and will create large component
